### PR TITLE
Use redux store for edit series modal

### PR DESCRIPTION
**Pull request by fearnlj01**
As TODO'd in PR #869

### CODE REVIEW

**Comment by fearnlj01** [comment]
I was going to have the setSeriesId action also set the open status of the modal at the same time... But that wasn't how it was done in the `browseFolder` modal, so I've kept things consistent instead 👍

**Review by harshithmohan — CHANGES_REQUESTED** [review]
We don't need to be consistent with browseFolderModal. That has a show parameter because it has nothing else. Here `show` can be just `seriesId !== -1`. We can make do with a single value instead of `show` and `seriesId`

File: src/components/Collection/Series/EditSeriesModal.tsx [hunk 1]
```diff
@@ -48,18 +48,17 @@ const EditSeriesModal = () => {
...
 
   const [activeTab, setActiveTab] = useState('name');
 
   return (seriesId === -1)
     ? null
     : (
-      <ModalPanel show={show} onRequestClose={onClose} header="Edit Series" size="md" noPadding noGap>
+      <ModalPanel show={seriesId !== -1} onRequestClose={onClose} header="Edit Series" size="md" noPadding noGap>
         <div className="flex h-[26rem] flex-row gap-x-6 p-6">
           <div className="flex shrink-0 gap-y-6 font-semibold">
```
Comment (harshithmohan) [inline src/components/Collection/Series/EditSeriesModal.tsx]: Remove the `(seriesId === -1) ? null :` part otherwise modal transition won't work

**Review by harshithmohan — CHANGES_REQUESTED** [review]

File: src/components/Collection/Series/EditSeriesModal.tsx (hunk 1 above)
Comment (fearnlj01) [reply src/components/Collection/Series/EditSeriesModal.tsx]: Went mad trying to work out which transition doesn't work as it looked like it was... Took a dozen mini-recordings for me to realise it was the exit transition 🙃

Fixed in the latest commit 👍

**Review by harshithmohan — APPROVED** [review]

**Pull request merged**